### ISSUE TITLE
NPE in DataUtility.java line 35

### ISSUE THREAD

**Issue by amaysakh** [awslabs/kinesis-kafka-connector#58]
ERROR WorkerSinkTask{id=XXX} Task threw an uncaught and unrecoverable exception. Task is being killed and will not recover until manually restarted. Error: null (org.apache.kafka.connect.runtime.WorkerSinkTask:607)
java.lang.NullPointerException
        at com.amazon.kinesis.kafka.DataUtility.parseValue(DataUtility.java:35)

**Comment by erindru** [comment]
Yes, you are right. The fact that JsonConverter requires a schema and the whole thing is expected to crash without one was not obvious to me at the time.

Since then I have encountered a bunch of other issues (the main one being something crashing internally when this connector is under load and it just hangs with no log output, I could never figure out what) so I stopped using it and wrote my own
